index.html

```diff
@@ -129,16 +123,21 @@
           </a>
         </div>
       </section>
-      <section id="interaction" class="interaction">
-        <div class="interaction__side-title">
+      <section id="interaction" class="interaction container container_row">
+        <div class="container__side-title container__side-title_light">
           <svg height="5rem" width="2rem">
             <rect height="4rem" width="0.07rem" x="0.95rem" y="0.5rem" fill="white" />
             <circle r="0.4rem" cx="1rem" cy="0.5rem" fill="white" />
           </svg>
           <span>Interaction</span>
         </div>
         <div class="interaction__info-container">
-          <div class="info-container__title">Interact with thousand of audience in no time</div>
+          <h2 class="info-container__title">
+            <span class="info-container__title_line">Interact with</span>
```

Comment:
You can put all the text inside `<h2>` and just limit it's width to make it fit multiple lines. No matter if those will not match the design exactly.